webapp/client/src/pages/PersonBHasDisabilityPage.js

```diff
@@ -0,0 +1,75 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+import FormHeader from "../components/FormHeader";
+import FormFieldRadio from "../components/FormFieldRadio";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import Details from "../components/Details";
+import { fieldPropType } from "../lib/propTypes";
+
+export default function PersonBHasDisabilityPage({
+  form,
+  fields,
+  prevUrl,
+  stepHeader,
+}) {
+  const { t } = useTranslation();
+
+  const tbold = function (key) {
+    return <Trans t={t} i18nKey={key} components={{ bold: <b /> }} />;
+  };
+
+  return (
+    <>
+      <StepHeaderButtons url={prevUrl} />
+      <FormHeader
+        title={stepHeader.title}
+        intro={tbold("lotse.hasDisability.intro_3")}
+      />
```

Comment:
This is a bit weird to me. This has no logic to it that title is set in the backend while intro is set in the frontend. I know, this is because of our weird hybrid model...Do you think it makes sense to just set everything in the frontend - including the title - and don't mind our hybrid-thing?

Reply:
I talked to Julian about it and unfortunately the backend still uses StepHeader.Title (had everything in the frontend before).

Reply:
Ah I see..ah well. 🤷